src/routes/Signup.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export default class Signup extends React.Component {
+  // TODO add redirection to "/" in case a user is already registered
+  render() {
+    return <div>Registration page</div>;
```

Comment:
Probably this component should render previous version of SignUp component (Same as Signin component)?

Reply:
We don't yet have a registration page component, do we? I created a separate task for this (WNB-124).

Reply:
hmm... I see you implemented registration form in your previous pull, but when merging, I seem to have somehow lost it... I will restore it

Reply:
I probably see why this happened.. You named your component file SignUp but named the function SignIn and exported it as SignIn https://github.com/Wannabook/wannabook-frontend/blob/develop/src/routes/SignUp.js#L90. This is why during conflict resolution I probably lost your registration component entirely

Reply:
Yes! You are right. I have to be more careful)